tests/core/test_examples.py

```diff
@@ -25,15 +28,22 @@ async def test_moodbot_example(unpacked_trained_moodbot_path):
 
 async def test_formbot_example():
     sys.path.append("examples/formbot/")
+    project = Path("examples/formbot/")
```

Comment:
somewhat curious whether this is gonna work on Windows 😆

Reply:
i assume since we have windows tests it does?